dll/directx/wine/ddraw/utils.c

```diff
@@ -580,8 +580,17 @@ unsigned int wined3dmapflags_from_ddrawmapflags(unsigned int flags)
         wined3d_flags |= WINED3D_MAP_NO_DIRTY_UPDATE;
     flags &= ~(handled | DDLOCK_WAIT | DDLOCK_READONLY | DDLOCK_NODIRTYUPDATE);
 
+#ifndef __REACTOS__
     if (flags)
         FIXME("Unhandled flags %#x.\n", flags);
+#elif defined(__REACTOS__) && defined(DBG)
+    if (flags == 0x20)
+    {
+        static int bWarnedOnce = 0; if (!bWarnedOnce) { bWarnedOnce++; FIXME("Unhandled flag 0x20 once\n"); }
```

Comment:
```suggestion
#elif defined(DBG)
    if (flags == DDLOCK_WRITEONLY)
    {
        static int bWarnedOnce; if (!bWarnedOnce) { bWarnedOnce++; FIXME("Unhandled flag DDLOCK_WRITEONLY once\n"); }
``````

Reply:
sure, I can do that. Initially wanted to keep the text within the fixme as shot as possible. But can live with your proposal.

Reply:
superseded